src/client.ts

```diff
@@ -94,6 +111,21 @@ export class Client {
    */
   private firefox_profile_path: string | undefined = undefined;
 
+  /**
+   * The file format in which the client will save the screenshot.
+   */
+  private screenshot_format: "jpeg" | "png" | "webp";
+
+  /**
+   * The image quality of screenshots (JPEG only)
+   */
+  private screenshot_quality: number;
+
+  /**
+   * The folder to store screenshots in
+   */
+  private screenshot_folder: string | null;
+
```

Comment:
I don't think we need this, i think these options should be delegated within the screenshot function as it's specifically related to taking a screenshot and not the client itself